modules/cards/card_1.html

```diff
@@ -0,0 +1,79 @@
+
+    <div class="card">
+        <div class="image">
+            <img src="images_and_icons/jonathan-petersson-gQhWMkYh3Yc-unsplash.jpg" alt="" class="profile-img">
+        </div>
+        <div class="text-data">
+            <h3>Equilibrium #3429</h3>
+            <p>Our Equlibrium collection promotes
+              balance and calm.</p>
+              
+            <h4>&#169; Creation of Harsh Shukla</h4>
+        </div>
+    </div>
+    <style>
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Poppins',sans-serif;
+}
+body{
```

Comment:
This style would affect the entire project.

Reply:
okay . I'll make the changes